minari/dataset/minari_storage.py

```diff
@@ -136,13 +202,7 @@ def update_episode_metadata(
         Warning:
             In case metadatas and episode_indices have different lengths, the longest is truncated silently.
         """
-        if episode_indices is None:
-            episode_indices = range(self.total_episodes)
-
-        with h5py.File(self._file_path, "a") as file:
-            for metadata, episode_id in zip(metadatas, episode_indices):
-                ep_group = file[f"episode_{episode_id}"]
-                ep_group.attrs.update(metadata)
+        ...
```

Comment:
If the proposed modification to `_HDF5Storage.update_episode_metadata()` is made, this warning just above this line is no longer needed.